apps/nextjs/.eslintrc.cjs

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
@juliusmarminge I omitted this file as it's currently simply extending the agreeto config.

From [turborepo docs](https://turbo.build/repo/docs/getting-started/create-new#understanding-eslint-config-custom:~:text=The%20way%20ESLint,the%20root%20instead.):
> The way ESLint finds its configuration file is by looking at the closest .eslintrc.js. If it can't find one in the current directory, it'll look in the directory above until it finds one.

> So that means that if we're working on code inside packages/ui (which doesn't have a .eslintrc.js) it'll refer to the root instead.

Do you consider it a best-practice to explicitly define the .eslintrc file in each app?

Reply:
True, we can remove it.

Reply:
@p6l-richard The only way I can get the typescript eslint parser happy is with that file there, I remember struggling with this before and I think this is the "easiest" solution. Without that file it tries locating the tsconfig in other places or something...

Maybe we can look into it later but for now I think this is fine.

Error without the explicit file + a `tsconfig.eslint.json` that tells it to parse all files: https://app.warp.dev/block/BLqxEGQyb4FMMUrxEtebTl

Reply:
Alright, without looking into it I don't myself -- didn't have any issues omitting this file.

Keeping it is fine for me. 👍

Reply:
> Alright, without looking into it I don't myself -- didn't have any issues omitting this file.
> 
> Keeping it is fine for me. 👍

yeah you didn't have the typescript eslint plugin and parser before, but it's very useful although configuring sucks :/